deprecation notice (#3)
gnosischain/prysm-client: README.md

# DEPRECATED
> [!WARNING]  

> > This is no longer supported as native Gnosis Chain support has been introduced. Please consider using the interactive guide provided in [documentation](https://docs.gnosischain.com/node/manual/).

Author: ```suggestion
> This repository is no longer supported. Please use the [official documentation](https://docs.gnosischain.com/node/manual/).
```
